SwiftCheck/TestOperators.swift

```diff
@@ -117,4 +117,4 @@ public func ^||^(p1 : Testable, p2 : Testable) -> Property {
 	return disjoin(p1.property, p2.property)
 }
 
-import func XCTest.XCTFail
+@exported import XCTest
```

Comment:
Probably.  It lets you export another framework's symbols as though they were yours.